apps/web/src/views/GaugesVoting/components/Table/AddGauge/AddGaugeModal.tsx

```diff
@@ -42,6 +43,10 @@ export const AddGaugeModal = ({ isOpen, onDismiss, selectRows, onGaugeAdd }) =>
   const totalGaugesWeight = useGaugesTotalWeight()
   const { data: gauges } = useGauges()
   const { filterGauges, setSearchText, filter, setFilter, setSort } = useGaugesFilter(gauges)
+  const dismissHandler = useCallback(() => {
+    setSearchText('')
+    onDismiss()
+  }, [])
```

Comment:
`onDismiss` should in deps

Reply:
added. onDismiss is required so no undefined check here.